README.md

```diff
@@ -270,7 +270,7 @@ directory.
   }
   ```
 
-  In the example you can also see another convenience property `actions` with all the html-reporter **Redux** actions.
+  In the example you can also see another convenient properties `actions` and `selectors` with all the html-reporter **Redux** actions and selectors.
```

Comment:
Maybe let's identify that `selectors` it's a memoized selectors which created using `reselect` library?